src/test/resources/ovs/v2/EventSchema.json

```diff
@@ -280,7 +280,7 @@
         },
         "publisher": {
           "description": "refers to a company or a legal entity.",
-          "type": "object",
+          "type": ["array","null","object"],
```

Comment:
Does this mean that arrays are supported? ...because they don't ;-)

Reply:
I agree, will fix this now.